directories/src/redux/reducers/pow-reducers.js

```diff
@@ -1,8 +1,6 @@
-import {
```

Comment:
this pow-reducers.js file is temporary BTW, it's just used to display the progress while we still have the luxuries of using React